Add OCCI infrastructure target (issue #599)

gibello opened the pull request:
First version, prototype.

vincent-zurczak commented:
None of your builds passed (dependency convergence).
Please, update your POM dependencies.

gibello commented:
Fixed, the build passes with tests. Removed dependencies on XML parsing 
and vijava IaaS APIs (not relevant for OCCI).

Le 12/10/2016 à 08:03, Vincent Zurczak a écrit :
>
> None of your builds passed (dependency convergence).
> Please, update your POM dependencies.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/roboconf/roboconf-platform/pull/702#issuecomment-253127115>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABLIf6QZUHirKHR8nDX44nYSTGEBe9Beks5qzHhCgaJpZM4KTvZR>.
>
